src/common/Loader/Loader.jsx

```diff
@@ -5,7 +5,7 @@ import { RotatingLines } from 'react-loader-spinner';
 
 import { LoaderWrap } from './Loader.styled';
 
-const Loader = ({ className }) => {
+const Loader = ({ width = '100', className }) => {
```

Comment:
Best practice:
Variables with default assignment should be settled after usual variables.

Reply:
The order of the variables has been changed